tests/optimize/test_rotosolve.py

```diff
@@ -492,9 +522,8 @@ def test_multiple_steps(self, qnode, param, num_freq, optimizer, optimizer_kwarg
             param = opt.step(
                 qnode,
                 *param,
-                num_freqs=num_freq,
-                optimizer=optimizer,
-                optimizer_kwargs=optimizer_kwargs,
+                nums_frequency=nums_frequency,
+                spectra=spectra,
```

Comment:
these tests only had minimal changes, which is a great vote of confidence in the logical changes!

Reply:
Yes, mostly the `optimizer` and `optimizer_kwargs` were moved to the Rotosolve initialization, `num_freqs` was renamed into `nums_frequency` (note that a different structure is expected for this kwarg now, though) and `spectra` as well as `shifts` was added.